src/util.cpp

```diff
@@ -1446,11 +1448,20 @@ string FormatFullVersion()
 
 #endif
 
+double Round(double d, int place)
+{
+    const double accuracy = place != 0
+            ? std::pow(10, place)
+            : 1;
```

Comment:
Oh I didn't know that. I think that 6.0 rounded to -1 is exactly what we want. In that case, can we just shorten it to: `const double accuracy = std::pow(10, place);`?